openfe/setup/alchemical_network_planner/relative_alchemical_network_planner.py

```diff
@@ -0,0 +1,437 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+import abc
+import copy
+from typing import Iterable, Callable, Type
+
+from gufe import (
+    Protocol,
+    AlchemicalNetwork,
+    LigandAtomMapping,
+    Transformation,
+    ChemicalSystem,
+)
+from gufe import SmallMoleculeComponent, ProteinComponent, SolventComponent
+
+
+from .abstract_alchemical_network_planner import (
+    AbstractAlchemicalNetworkPlanner,
+)
+
+from .. import LomapAtomMapper
+from ..ligand_network import LigandNetwork
+from ..atom_mapping.ligandatommapper import LigandAtomMapper
+from ..atom_mapping.lomap_scorers import default_lomap_score
+from ..ligand_network_planning import generate_minimal_spanning_network
+from ..chemicalsystem_generator.abstract_chemicalsystem_generator import (
+    AbstractChemicalSystemGenerator,
+)
+from ..chemicalsystem_generator import (
+    EasyChemicalSystemGenerator,
+    RFEComponentLabels,
+)
+from ...protocols.openmm_rbfe.equil_rbfe_methods import RelativeLigandProtocol
+
+
+# TODO: move/or find better structure for protocol_generator combintations!
+PROTOCOL_GENERATOR = {
+    RelativeLigandProtocol: EasyChemicalSystemGenerator,
+}
+
+
+"""
+    easy relative campaigner
+"""
+
+
+class RelativeAlchemicalNetworkPlanner(
+    AbstractAlchemicalNetworkPlanner, abc.ABC
+):
+
+    """
+    magics
+    """
+
+    _chemical_system_generator: AbstractChemicalSystemGenerator
+
+    """
+        public funcs
+    """
+
+    def __init__(
+        self,
+        name: str = "easy_rfe_calculation",
+        mappers: Iterable[LigandAtomMapper] = [LomapAtomMapper()],
+        mapping_scorer: Callable = default_lomap_score,
+        ligand_network_planner: Callable = generate_minimal_spanning_network,
+        protocol: Protocol = RelativeLigandProtocol(
+            RelativeLigandProtocol._default_settings()
+        ),
+    ):
+        """a simple strategy for executing a given protocol with mapper, mapping_scorers and networks for relative FE approaches.
```

Comment:
```suggestion
        """A simple strategy for executing a given protocol with mapper, mapping_scorers and networks for relative FE approaches.
```
Also something something 79 characters